gocron.go

```diff
@@ -127,6 +127,10 @@ func getFunctionName(fn interface{}) string {
 	return runtime.FuncForPC(reflect.ValueOf(fn).Pointer()).Name()
 }
 
+func getFunctionNameOfPointer(fn interface{}) string {
```

Comment:
hey @ErfanMomeniii thanks for the thorough contribution! 

Could you help me understand what the use-case is for passing a pointer to a function into the scheduler?